GVFS/GVFS.Service/Mac/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using GVFS.Common;
+using GVFS.Common.Tracing;
+using GVFS.PlatformLoader;
+using System;
+
+namespace GVFS.Service
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            GVFSPlatformLoader.Initialize();
+
+            AppDomain.CurrentDomain.UnhandledException += UnhandledExceptionHandler;

```

Comment:
I think AppDomain is Framework only and didn't make it into .NET Core?

Reply:
I believe it made it over in .NET Core 2.0: https://docs.microsoft.com/en-us/dotnet/api/system.appdomain.unhandledexception?view=netcore-2.0

Reply:
Right - its available in the [dotnet core sources](https://github.com/dotnet/corefx/find/master).

We are also using it in `GVFS.Mount.Mac::InProcessMountVerb::Execute()`